lib/galaxy/objectstore/__init__.py

```diff
@@ -220,6 +220,16 @@ def from_xml(clazz, config, config_xml, **kwd):
         config_dict = clazz.parse_xml(config_xml)
         return clazz(config, config_dict, **kwd)
 
+    def to_dict(self):
+        extra_dirs = []
```

Comment:
Do you think there'd be an advantage of listing attributes that should be serialised as a class variable ?
i.e 
```Python
class SomeObjectStore(object):

    serializable_attributes = ('config', 'type', 'extra_dirs',)

    def to_dict(self):
        return {attr: getattr(attr, self) for attr in self.serializable_attributes}
```

Reply:
I'll consider this - I'm not sure it would simplify much in the subclasses. It would be closer to what we do other places in the app though I'll ready admit. Thanks for the review!